ggshield/cmd/auth/login.py

```diff
@@ -189,7 +190,14 @@ def token_login(config: Config, instance: Optional[str]) -> None:
 
     # enforce using the token (and not use config default)
     client = create_client(api_key=token, api_url=config.api_url)
-    response = client.get(endpoint="token")
+    try:
+        response = client.get(endpoint="token")
+    except requests.exceptions.ConnectionError as e:
+        if "Failed to resolve" in str(e):
```

Comment:
I think we could be even more precise by searching `Failed to resolve '{instance}'` ?

Reply:
not this easily, for example with `$GITGUARDIAN_INSTANCE='https://dashboard.example.com'`, the error would be `Failed to resolve 'dashboard.example.com'` so I would have to crop the instance variable to look for it... I'm afraid to introduce more unwanted behavior. 
Do you think I should still do it ?